cpp/lib/include/FileUtil.h

```diff
@@ -503,6 +504,12 @@ bool Copy(const std::string &from_path, const std::string &to_path, const bool t
           const loff_t offset = 0, const int whence = SEEK_CUR);
 void CopyOrDie(const std::string &from_path, const std::string &to_path);
 
+/**
+ * \brief Copy file that support cross filesystem's type. The maximum size of copy file is 2GB only.
```

Comment:
Now the implementation uses cp, so the size limitation can be removed from the comment.